contracts/eosio.system/src/eosio.system.cpp

```diff
@@ -407,20 +397,20 @@ namespace eosiosystem {
         res.cpu_weight = asset( 0, system_contract::get_core_symbol() );
       });
 
-      set_resource_limits( newact.value, 0, 0, 0 );
+      set_resource_limits( newact, 0, 0, 0 );
    }
 
-   void native::setabi( const name& acnt, const std::vector<char>& abi ) {
+   void native::setabi( const name& account, const std::vector<char>& abi ) {
```

Comment:
change these back to `acnt` these are a part of the interface, so these would result in breaking changes to dapps.